open_event/api/utils.py

```diff
@@ -1,5 +1,10 @@
 from flask_restplus import Model, fields, reqparse
 
+# Parameters for a paginated response
+PAGE_PARAMS = {
+    'start': {'description': 'ID to begin from', 'type': int},
```

Comment:
`start` is not the ID but the position. Say if event 3 has sessions with ids 3,5,7,9,10 then for `start=3` everything after session_id=7 in event 3 is elligible for return.